### PR TITLE
GeoNode project with geoprocessing code for ForecastIO air temperature

### PR DESCRIPTION
**Pull request by mbertrand**
This is a fork of the standard GeoNode project template (https://github.com/GeoNode/geonode-project) that contains code for processing ForecastIO 's hourly global air temperature data and NASA's Global Flood Monitoring System (GFMS) data.  Instructions for setting up are in the README.rst file.  A sample configuration file for running celery under supervisor is also included (geonodegp/celery.conf).

### CODE REVIEW

**Comment by mgrauer** [comment]
@jbeezley Just wanted to make sure you were aware of this.

**Comment by aashish24** [comment]
@jbeezley can you review this? 

@mbertrand thanks!

File: geonode/geonodegp/data_queues/processors.py [hunk 1]
```diff
@@ -0,0 +1,987 @@
...
+from geonodegp.data_queues.csv_helpers import UnicodeWriter, UnicodeReader
+from geonodegp.data_queues.helpers import get_band_count, gdal_translate, cdo_invert, postgres_query, ogr2ogr_exec
+from geonodegp.settings import DEFAULT_WORKSPACE
+from geonode.geoserver.helpers import ogc_server_settings, OGC_Servers_Handler
+from geonode.geoserver.management.commands.updatelayers import Command as UpdateLayersCommand
+import shutil
+
+logger = logging.getLogger("geonodegp.data_queues.processors")
+
+class GeoDataProcessor(object):
```
Comment (jbeezley) [inline geonode/geonodegp/data_queues/processors.py]: @kotfic  This is a nice set of classes for doing real workflow tasks that could be useful as an example when working on the gaia analysis library.

**Comment by jbeezley** [comment]
We will want to add some testing and devops stuff to make deployment easier once we have a better idea for how this will be integrated into the analysis/workflow library.  (@mbertrand What do you use for provisioning in vagrant?)  Until then, this seems okay to store here temporarily.  We can always subtree it out if necessary.

**Comment by mbertrand** [comment]
@jbeezley I've just been running this directly in OSX without using vagrant.

**Comment by jbeezley** [comment]
Okay, I assumed the `/home/vagrant` in celery config was meaningful.

**Comment by mbertrand** [comment]
@jbeezley, sorry, I misunderstood the question.  I actually did run this code in a vagrant vm to test it out before checking in and make sure it works in that environment, which is where that celery.config file comes from (I included it as an example).  It was an ubuntu/trusty64 image and I installed dependencies manually.  But most of the time I just run GeoNode in development mode directly on my laptop without a VM.

**Comment by jbeezley** [comment]
No problem, I was just curious.

**Comment by jbeezley** [comment]
@mbertrand Thanks for pushing this, it is all fine from your end.  We decided that I will take over here and add some devops magic before we merge it in.

**Comment by jbeezley** [comment]
I'm moving this to a new branch so I can amend the PR.

**Pull request closed without merging**